<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Overcoming the False Trade-off in Genomics | Shuvom Sadhuka </title> <meta name="author" content="Shuvom Sadhuka"> <meta name="description" content="Does privacy have to be at odds with collaboration in biomedical research?"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%8C&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://shuvom-s.github.io/blog/2023/overcoming-the-false-tradeoff-in-genomics/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Shuvom Sadhuka </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Overcoming the False Trade-off in Genomics</h1> <p class="post-meta"> Created in June 01, 2023 </p> <p class="post-tags"> <a href="/blog/2023"> <i class="fa-solid fa-calendar fa-sm"></i> 2023 </a>   ·   <a href="/blog/tag/genomics"> <i class="fa-solid fa-hashtag fa-sm"></i> genomics</a>   <a href="/blog/tag/privacy"> <i class="fa-solid fa-hashtag fa-sm"></i> privacy</a>   <a href="/blog/tag/law"> <i class="fa-solid fa-hashtag fa-sm"></i> law</a>   ·   <a href="/blog/category/blog-posts"> <i class="fa-solid fa-tag fa-sm"></i> blog-posts</a> </p> </header> <article class="post-content"> <div id="markdown-content"> <p>Note: This post was originally published for MIT’s <a href="https://computing.mit.edu/wp-content/uploads/2023/06/Overcoming.pdf" rel="external nofollow noopener" target="_blank">Envision the Future of Computing Prize</a>, where it won an honorable mention.</p> <p>On June 26, 2000, President Bill Clinton and Prime Minister Tony Blair jointly announced to the world that the first draft of the human genome had been completed. Speaking with unfettered optimism on the implications of Human Genome Project (HGP), President Clinton <a href="https://archive.nytimes.com/www.nytimes.com/library/national/science/062700sci-genome-text.html" rel="external nofollow noopener" target="_blank">declared</a>:</p> <blockquote> <p>“Without a doubt, this is the most important, most wondrous map ever produced by humankind. […] It will revolutionize the diagnosis, prevention, and treatment of most, if not all, human diseases. […] In fact, it is now conceivable that our children’s children will know the term cancer only as a constellation of stars.”</p> </blockquote> <p>Two decades later, the president’s lofty promises have proven to be somewhat prescient. There is little doubt that whole genome sequencing — now more than five orders of magnitude cheaper than in 2000 — has revolutionized biomedicine.</p> <p>Pharmacogenomics is already empowering precision medicine targeted to the genetic mutations a patient carries. While a blanket cure for cancer as President Clinton envisioned is yet to be realized, CRISPR gene editing technologies have also generated considerable scientific excitement for the next era of therapeutic medicine. Even long-held anthropological questions regarding human migration can be <a href="https://www.nytimes.com/2018/04/20/books/review/david-reich-who-we-are-how-we-got-here.html" rel="external nofollow noopener" target="_blank">answered</a> through computational analysis of ancient DNA. These innovations are not limited to academia either. Today, there are numerous publicly traded companies (e.g. <a href="https://www.23andme.com/" rel="external nofollow noopener" target="_blank">23andMe</a>) whose core products involve sequencing or analysis of genomic data.</p> <p>Each of these efforts has <em>only</em> been made possible through large international collaborations since the HGP, and further breakthroughs in genomics will inevitably continue to change the pace of drug discovery and biomedical innovation. Indeed, collaboration is at the heart of genomics and biomedical research: groundbreaking discoveries only occur when data is pooled across multiple ethnicities, conditions, and backgrounds.</p> <p>However, buried within Clinton’s speech were other concerns about the future of genomics:</p> <blockquote> <p>“As we unlock the secrets of the human genome, we must work simultaneously to ensure that new discoveries never pry open the doors of privacy. And we must guarantee that genetic information cannot be used to stigmatize or discriminate.”</p> </blockquote> <p>President Clinton’s words were especially important given the historically fraught relationship between population genetics and minority communities. Some of the original titans of the field and inventors of mathematical tools for genetic analysis manipulated their inventions to promote scientific racism. Absent proper safeguards, access to the highest resolution information about human ancestry, DNA, can be used for novel forms of discrimination and surveillance.</p> <p>As international collaborations explode in size, concerns regarding genomic privacy and ethics are growing; in the past two years, US Congress has debated the federal <a href="https://www.rubio.senate.gov/public/index.cfm/2021/5/rubio-introduces-bills-to-counter-growing-threat-of-china-s-collection-of-american-s-genomic-data" rel="external nofollow noopener" target="_blank">Genome Data Security Act</a> and California has signed the <a href="https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=202120220SB41" rel="external nofollow noopener" target="_blank">Genetic Information Privacy Act</a> into law. While these laws aim to protect genomic data, it is unclear exactly what kind of attacks and analyses may be possible in the event of a future privacy breach. As our understanding of the human genome continues to evolve, richer biometric information can be extracted from genomic samples. For instance, prenatal genetic testing can <a href="https://www.acog.org/womens-health/faqs/prenatal-genetic-screening-tests" rel="external nofollow noopener" target="_blank">illuminate</a> disease risks prior to birth.</p> <p>However, we do know two key differences in genomic privacy and ethics concerns compared to other forms of data: (1) unlike credit card numbers or social media, the human genome is <em>immutable</em>, so a one-time leak of information is a lifetime leak, (2) genomes are strongly correlated between relatives, so a leak has privacy implications beyond the individual, for families and even communities.</p> <p>Traditionally, stronger privacy regulations are considered at odds with collaborative research and development. However, <strong>this is simply not an option in genomics</strong>, where strong restrictions will prevent life-saving therapeutics from being developed.</p> <p>In this essay, I argue for the need to develop new technologies that both enhance genomic privacy <em>and</em> foster large international collaborations. Specifically, I argue that the oft-repeated trade-off between privacy and utility is a false dichotomy that can be overcome in genomics with significant engineering and legal effort. We must develop two forms of data security and privacy to enable such collaborations:</p> <ol> <li> <strong>Institutional data security</strong>, or security of large-scale biological data repositories, hospitals, and corporations against malicious actors hoping to steal or manipulate this data.</li> <li> <strong>End-user data privacy</strong>, or guarantees to patients participating in studies that their individual data will not be identifiable or inappropriately shared.</li> </ol> <p>Importantly, strictly “legislating” around this privacy issue to institute tighter data access measures will only slow the pace of research and play into the trade-off. Instead, we must lean on new computing technologies to maintain scientific efficiency while promoting security, privacy, and compliance.</p> <h2 id="a-new-era-of-collaborative-research">A New Era of Collaborative Research</h2> <p>Collaborative scientific research is a prerequisite for genomics. Genetic signatures of diabetes, for example, can differ from population to population, so generalizable and statistically significant findings require pooling data from organizations worldwide. This is reflected in the endlessly growing list of consortia dedicated to the study of biomedical questions through genomics: <a href="https://www.ukbiobank.ac.uk/" rel="external nofollow noopener" target="_blank">UK Biobank</a>, <a href="https://allofus.nih.gov/" rel="external nofollow noopener" target="_blank">NIH All of Us</a>, <a href="https://www.finngen.fi/en" rel="external nofollow noopener" target="_blank">FinnGen</a>, and so on.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/genomecost-480.webp 480w,/assets/img/genomecost-800.webp 800w,/assets/img/genomecost-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/genomecost.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> The cost of sequencing a human genome has dropped precipitously. Large databases now contain thousands of genomes. (Figure credit: NHGRI) </div> <p>The change is being acknowledged at all levels of biomedical research. President Biden’s cancer moonshot goal — started in his Vice President years — to reduce the rate of cancer deaths by half in twenty-five years includes a project to develop a cloud-based cancer genomic data analysis platform authored by Microsoft, Amazon Web Services, and the National Cancer Institute. Closer to MIT, the Broad Institute is partnering with Microsoft to develop <a href="https://www.microsoft.com/en-us/research/blog/biomedical-research-platform-terra-now-available-on-microsoft-azure/" rel="external nofollow noopener" target="_blank">Terra</a>, a secure and efficient cloud-based biomedical data analysis platform.</p> <p>Researchers, clinicians, and industry partners are using these databases and platforms to improve precision medicine. For example, the Cancer Genome Atlas has gathered more than 2.5 petabytes of data, which has been used to identify key cancer genes like BRCA1 and provide more fine-grained diagnoses beyond the standard stages of cancer.</p> <p>Kemal Malik, director of innovation for Bayer, <a href="https://www.nationalgeographic.com/science/article/partner-content-genomics-health-care#:~:text=Genomics%2C%20the%20study%20of%20genes,as%20height%20and%20hair%20color." rel="external nofollow noopener" target="_blank">highlights</a> the crucial role genomics plays in precision medicine and disease treatment:</p> <blockquote> <p>“The Holy Grail in health care has long been […] precision medicine. But getting to the level of precision we wanted wasn’t possible until now. What’s changed is our ability to sequence the human genome […] To date, genomics has had the most impact on cancer because we can get tissue, sequence it, and identify the alterations […] In the future we’ll see every cancer patient sequenced, and we’ll develop specific drugs to target their particular genetic alteration.”</p> </blockquote> <p>Indeed, the <em>key</em> driver of precision medicine has been pharmacogenomics. Under this new clinical paradigm, drugs are prescribed only if the predicted patient response (based on genomic information) is positive. Pharmacogenomics is not a pipe dream: since 2011, the FDA has <a href="https://www.fda.gov/media/124784/download" rel="external nofollow noopener" target="_blank">labeled</a> 250+ prescriptions with dosage recommendations based on genetic differences. For instance, the breast cancer drug PIQRAY was shown to be more effective in patients with a mutation to the PIK3CA gene than those without one. As we develop a more nuanced understanding of pharmacogenomics, it’s possible — even likely — that we will move away from FDA-curated labels to FDA-approved drug recommendation software. Some <a href="https://genomind.com/" rel="external nofollow noopener" target="_blank">startups</a> are already <a href="https://genesight.com/" rel="external nofollow noopener" target="_blank">testing</a> these <a href="https://www.genxys.com/" rel="external nofollow noopener" target="_blank">waters</a>.</p> <p>In a related development, CRISPR technologies are moving the needle on gene editing, which could help cure once incurable diseases. After a series of publications in 2012 demonstrating that CRISPR/Cas9 could edit genes through precise cuts and natural repairs to DNA sequences, CRISPR has captured the imagination of biotechnologists worldwide. Vertex Pharmaceuticals has already <a href="https://www.nytimes.com/2022/12/09/opinion/crispr-gene-editing-cures.html" rel="external nofollow noopener" target="_blank">cured</a> 31 people of sickle cell disease, and other companies are testing preclinical drugs. Scientists have also been using CRISPR to develop tests for infectious diseases at lower costs than PCR tests and even extreme-weather resistant crops.</p> <p>Genomics has even answered questions in social sciences. Geneticists like Svante Pääbo (2022 Nobel Laureate) have leveraged computational techniques to unlock age-old questions about human migration. By mapping and comparing fragments of DNA found in ancient human bones, they have been able to <a href="https://www.nytimes.com/2018/04/20/books/review/david-reich-who-we-are-how-we-got-here.html" rel="external nofollow noopener" target="_blank">establish</a> new lineages in the human story, like the archaic humans from the Denisovan caves, whose genomes account for up to 5% of some modern lineages.</p> <h2 id="characterization-of-privacy-risks">Characterization of Privacy Risks</h2> <p>In parallel with the scientific advances, the call for genomic privacy has grown from a mellow hum to a thunderous roar. It is natural to believe that the path forward for genomics is to remove barriers to collaboration, a route which has already provided numerous discoveries. However, without proper safeguards, the rift of trust between the public and medical science will expand. One can only imagine if baseless conspiracies — such as <a href="https://www.unicef.org/montenegro/en/stories/covid-19-vaccine-does-not-change-human-dna" rel="external nofollow noopener" target="_blank">claims</a> that COVID vaccines directly edit human DNA — are provided genuine substance in the event of a large-scale genetic privacy breach. Maintaining trust is one of many concerns, and as we expand these collaborations public trust must be paramount.</p> <p>Early privacy studies indicated that relatively few — just 75 — genomic coordinates (“single nucleotide polymorphisms”) were needed to uniquely identify many individuals, but it was unclear how such findings would generalize given the sparsity of large-scale sequencing databases. Re-identifying individual genomes at scale seemed akin to searching for a genetic needle in a haystack of DNA.</p> <p>At some point in the 2010s, the tone shifted. A flurry of papers demonstrating that anonymized genomic data could be linked to individuals or other phenotypic information raised alarm. One landmark <a href="https://www.science.org/doi/full/10.1126/science.1229566" rel="external nofollow noopener" target="_blank">study</a> in 2018 leveraged paternal surname inheritance to link anonymous genomes to specific names. By exploiting the Y chromosome — present only in males and thus, like surnames, typically inherited from the father — and a small (private) database of labeled genomes, researchers were able to triangulate the identity of a substantial fraction of males. Extrapolating from their results, they estimated that 12% of all European descent males in the US were susceptible to such an attack. Responding to the work, Eric Green, former director of the NIH Human Genome Research Institute, <a href="https://www.abajournal.com/news/article/researchers_identify_dna_study_volunteers_through_online_sleuthing_consent_" rel="external nofollow noopener" target="_blank">noted</a>, “we are […] an awareness moment.”</p> <p>Since the publication of this and similar attacks, new avenues of privacy breaches have been demonstrated, both in real-world and sandboxed settings. The attacks run the gamut of techniques, but some noteworthy examples include:</p> <ol> <li> <p><a href="https://pubmed.ncbi.nlm.nih.gov/34746296/" rel="external nofollow noopener" target="_blank">Reconstruction</a> of private genomes by repeatedly sending (legal) queries to a server.</p> </li> <li> <p>A <a href="https://www.washingtonpost.com/politics/2022/07/21/hacks-genetic-firms-pose-risk-patients-experts-say/#:~:text=Since%20the%20beginning%20of%20last,of%20Health%20and%20Human%20Services." rel="external nofollow noopener" target="_blank">data breach</a> from servers of a company due to unencrypted data stored on a problematic server.</p> </li> </ol> <p>Unlike other forms of data, genomic data is immutable, so these attacks have implications well beyond the victim and the specific time of the attack. It is nearly impossible to think of another type of data which, if leaked, can reveal sensitive health information of the victim’s future great-grandchildren. Brad Malin, professor of computer science at Vanderbilt, <a href="https://www.washingtonpost.com/politics/2022/07/21/hacks-genetic-firms-pose-risk-patients-experts-say/" rel="external nofollow noopener" target="_blank">notes</a> that the risks are “highly dependent on how the adversary wants to use the data.” Feasible possibilities <a href="https://www.wired.com/story/the-us-urgently-needs-new-genetic-privacy-laws/" rel="external nofollow noopener" target="_blank">include</a> employment and housing discrimination based on genetics, which is legal in certain municipalities at the moment.</p> <p>In response, various government agencies have begun instituting tighter controls for genomic data. Last year, the National Institute of Standards and Technology (NIST) <a href="https://www.nccoe.nist.gov/projects/cybersecurity-and-privacy-genomic-data" rel="external nofollow noopener" target="_blank">announced</a> a new project to “identify genomic data cybersecurity and privacy concerns and develop guidance to address these challenges.” NIST’s project touches all levels of genomic analysis, from ensuring that sequencing devices themselves are secure to writing better access protocols for genomic data repositories.</p> <p>FBI agent Ed You <a href="https://www.ft.com/content/245a7c60-6880-11e7-9a66-93fb352ba1fe" rel="external nofollow noopener" target="_blank">notes</a> that “cross-border deals are not the only risks to US genetic data. The healthcare industry is notoriously vulnerable to cyberattacks.” These concerns have directly influenced legislation. <a href="https://www.techtarget.com/healthtechsecurity/news/366595025/Growing-Number-of-States-Enact-New-Genetic-Data-Privacy-Laws" rel="external nofollow noopener" target="_blank">Four states</a> have passed genetic privacy protections, and Congress is currently considering the Genomic Data Security Act, a bill which would place restrictions on Chinese access to American genomic data.</p> <h2 id="overcoming-the-privacy-utility-trade-off">Overcoming the Privacy-Utility Trade-off</h2> <h3 id="a-false-trade-off">A False Trade-off?</h3> <p>Typically, the tension between privacy and collaboration is framed as zero-sum. Former UK Health Secretary Matt Hancock bluntly <a href="https://www.gov.uk/government/speeches/we-must-tackle-the-serious-ethical-challenges-of-dna-analysis" rel="external nofollow noopener" target="_blank">noted</a> that: “it’s outrageous that too often, anonymised data […] can’t be used for research. We will unlock that data because […] it saves lives.” Hancock’s concerns are reasonable and highlight the cost of strict measures: patient lives. Nonetheless, the false dichotomy is dangerous and could lead us into a vicious cycle of removing access barriers only to later realize privacy issues. We must therefore overcome Hancock’s implicit belief that privacy and collaborative research are “too often” at odds.</p> <p>The previous attacks demonstrate a need for both <em>institutional data security</em> and <em>end-user genomic privacy</em> paradigmsto overcome this double bind. While the distinction is somewhat artificial, attacks stemming from unauthorized access fall more in an <em>institutional</em> oversight realm. On the other hand, reconstruction of a private genome through public queries seems to be an <em>end-user</em> privacy issue.</p> <p>The distinction is perhaps best explained by the services offered in the financial sector. If a bank liquidates, it is typically blamed for its poor <em>institutional</em> management. To compensate for this, <em>end users</em> can choose to only hold savings in FDIC-insured banks, so that their money is federally insured in case of mismanagement.</p> <p>Despite strict privacy regulations, a thriving market — fintech — exists to provide customers with better solutions and overcome a double bind. A similar balance is needed in genomics.</p> <h3 id="a-response-from-computing">A Response from Computing</h3> <p>Several computing technologies actively being developed at MIT could empower the secure and efficient genomic data collaboration needed to overcome the false trade-off between privacy and collaboration.</p> <p>On the institutional data security front, there are proof-of-concept and real-world applications of cryptographic tools to enable secure collaboration. As an instructive example, imagine there are three biobanks which would like to jointly analyze their data without revealing their private patient data.</p> <p>Secure multiparty computation (SMC) defines a set of protocols which enable joint computation of a function when the data is distributed across multiple parties. For example, if the function to compute is $F(x,y,z) = xyz$, then SMC allows the parties which separately own $x$, $y$, and $z$ to compute $F$ without revealing anything about their inputs. The key method behind SMC is known as secret sharing, which uses some elegant properties of polynomial interpolation to guarantee security.</p> <p>This is a promising paradigm for collaboration, as functions we would like to compute over distributed datasets could now be computed in cryptographically secure ways. <a href="https://www.nature.com/articles/nbt.4108" rel="external nofollow noopener" target="_blank">Researchers at MIT</a> have already shown proof-of-concept results for collaborative genomic analyses — such as gene-disease studies — using SMC without sharing raw patient data.</p> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/fhe-480.webp 480w,/assets/img/fhe-800.webp 800w,/assets/img/fhe-1400.webp 1400w," type="image/webp" sizes="95vw"></source> <img src="/assets/img/fhe.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> FHE allows direct computation on encrypted data. </div> <p>In a separate but similar setting, imagine a hospital holds a partial genomic sequence of a patient and would like to impute the rest. Genotype imputation is an expensive task, so the hospital asks a remote server to help with the imputation. Fully homomorphic encryption (FHE) allows direct computation on encrypted data. In this paradigm, the hospital <a href="https://www.sciencedirect.com/science/article/pii/S240547122100288X" rel="external nofollow noopener" target="_blank">sends</a> an encrypted genome to a remote server, which provides (an encrypted version of) the correct answer without learning the input.</p> <p>While FHE/SMC provide cryptographic security guarantees, engineering them to be efficient is challenging in practice. The additional memory requirements for these technologies makes computation exceedingly burdensome in many cases, and each line of the computer program further elongates the runtime. Limited network bandwidths can explode the total time required if the network traffic needed for the protocol is high. For these reasons, SMC/FHE are yet not used at scale in practice. These protocols are especially hard to implement for intrinsically resource-intensive computations like deep learning. We therefore <strong>must</strong> continue developing and deploying scalable SMC/FHE and related technologies to provide a bridge — cryptographically secure protocols for widespread collaboration — across the false trade-off between privacy and collaboration.</p> <p>On the <em>end-user</em> data front, methods like differential privacy are <a href="https://academic.oup.com/bioinformatics/article/36/6/1696/5614817" rel="external nofollow noopener" target="_blank">providing</a> stronger guarantees that individual patient data will not be leaked through some of the statistical attacks described earlier. Differential privacy — a technique that adds noise to any aggregated data releases — provides a mathematical framework to pinpoint the chance that a patient’s data can be re-identified from legal queries to database. Calibrating the correct noise to add is notoriously difficult: too little noise weakens privacy while too much noise can muddle important downstream analyses.</p> <p>The solution in applications thus far has been a tenuous consensus from experts and stakeholders. Even with such careful deliberations, institutions like the US Census have received <a href="https://www.washingtonpost.com/local/social-issues/2020-census-differential-privacy-ipums/2021/06/01/6c94b46e-c30d-11eb-93f5-ee9558eecf4b_story.html" rel="external nofollow noopener" target="_blank">criticism</a> for their noise levels.</p> <p>It is unlikely that any noise level — or any cryptographic protocol — will satisfy everyone’s preferred privacy level. Privacy is inherently personal: you may mind your friend snooping through your text messages, but your neighbor may not. The only truly trustworthy and long-term solution is to therefore offer fine-grained consent laws to users. These laws should enable users to specify who to share their data with and what analyses are allowed with their data. At the loosest level, no protections will be available beyond what is federally required. At the strongest level, the user can retract their data from a database and scrap any traces of it anywhere. An in-between level may include an authorization for use in collaborative studies, but only if technologies such as FHE/SMC are used.</p> <p>The specific formulations of consent will require the collaboration of patients, clinicians, researchers, and legislators, but the need for fine-grained privacy laws is clear.</p> <h3 id="a-look-to-the-future">A Look to the Future</h3> <p>Perhaps President Clinton will prove to be right, only several decades later. It is conceivable that certain rare diseases and cancers will only be historical afflictions at some point in the 21st century. As ambitious as that may seem, the 20th century brought miracle cures: a polio vaccine, the first antibiotics, and IVF babies. Unlocking the human genome will provide many more.</p> <p>As we look to this future with starry-eyed optimism, it is critical that we not get locked into an endless debate between privacy and utility. Instead, we must imagine new worlds — both technologically and through legislation — in which the two co-exist. Bridging the gap between privacy and collaboration is the only way forward to save as many lives as possible, and some of the ideas introduced here could lead the way. It is only through locking our genetic secrets that we can fully unlock our genome’s revolutionary potential.</p> </div> </article> <br> <hr> <br> <ul class="list-disc pl-8"></ul> <h2 class="text-3xl font-semibold mb-4 mt-12">Enjoy Reading This Article?</h2> <p class="mb-2">Here are some more articles you might like to read next:</p> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2025/measuring-entropy/">Measuring Entropy</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/advice-on-applying-to-fellowships/">Measuring Entropy</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2024/my-thoughts/">A running list of writing ideas</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2020/on-voting-paradoxes/">On Voting Paradoxes</a> </li> <li class="my-2"> <a class="text-pink-700 underline font-semibold hover:text-pink-800" href="/blog/2020/democracy-and-the-central-limit-theorem/">Democracy and the Central Limit Theorem</a> </li> <div id="giscus_thread" style="max-width: 930px; margin: 0 auto;"> <script>
      let giscusTheme = determineComputedTheme();
      let giscusAttributes = {
        src: 'https://giscus.app/client.js',
        'data-repo': 'shuvom-s/shuvom-s.github.io',
        'data-repo-id': 'R_kgDONxLE6Q',
        'data-category': 'General',
        'data-category-id': 'DIC_kwDONxLE6c4Cm6ys',
        'data-mapping': 'pathname',
        'data-strict': '0',
        'data-reactions-enabled': '1',
        'data-emit-metadata': '0',
        'data-input-position': 'bottom',
        'data-theme': giscusTheme,
        'data-lang': 'en',
        crossorigin: 'anonymous',
        async: '',
      };

      let giscusScript = document.createElement('script');
      Object.entries(giscusAttributes).forEach(([key, value]) => giscusScript.setAttribute(key, value));
      document.getElementById('giscus_thread').appendChild(giscusScript);
    </script> <noscript>Please enable JavaScript to view the <a href="http://giscus.app/?ref_noscript" rel="external nofollow noopener" target="_blank">comments powered by giscus.</a> </noscript> </div> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Shuvom Sadhuka. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>